### PR TITLE
Add checksum warning to readme

### PR DESCRIPTION
**Pull request by dabura667**
As discussed here

https://github.com/bitcoinjs/bip39/pull/24#issuecomment-144913036

### CODE REVIEW

**Comment by dcousens** [comment]
I'm not sure if the entire entry needs to be bold.
The advice is sound.

File: README.md [hunk 1]
```diff
@@ -6,6 +6,14 @@ BIP39
 
 JavaScript implementation of [Bitcoin BIP39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki): Mnemonic code for generating deterministic keys
 
+## Reminder for developers
+
+***Please remember to allow recovery from mnemonic phrases that have invalid checksums (or that you don't have the wordlist)***
+
+***When a checksum is invalid, warn the user that the phrase is not something generated by your app, and ask if they would like to use it anyway. This way, your app only needs to hold the wordlists for your supported languages, but you can recover phrases made by other apps in other languages.***
+
+***However, there should be other checks in place, such as checking to make sure the user is inputting 12 words or more separated by a space. ie. `phrase.trim().split(/\s+/g).length >= 12`***
```
Comment (dcousens) [inline README.md]: Will `phrase.trim().split(/\s+/g).length >= 12` work with Japanese mnemonics?

Comment (dabura667) [inline README.md]: Yes it will, \u3000 is contained in all regex versions of \s

**Comment by dcousens** [comment]
Thanks @dabura667

**Pull request merged**